lib/ex.coffee

```diff
@@ -169,17 +176,28 @@ class Ex
       trySave(-> saveAs(fullPath)).then(deferred.resolve)
 
     deferred.promise
+  # save all files function
+  saveAll: ->
+    atom.workspace.saveAll()
+  # save all then quit
+  saveAllThenQuit: -  >
+    atom.workspace.saveAll()
+    @quitAll()
```

Comment:
I've created this function will be referenced on shortcut command that I've added...